src/providers/codebuild.ts

```diff
@@ -274,7 +276,8 @@ export class CodeBuildRunnerProvider extends BaseProvider implements IRunnerProv
       buildSpec.phases.install.commands = [
         'cd \\actions',
         'if (${Env:RUNNER_VERSION} -eq "latest") { $RunnerFlags = "" } else { $RunnerFlags = "--disableupdate" }',
-        './config.cmd --unattended --url "https://${Env:GITHUB_DOMAIN}/${Env:OWNER}/${Env:REPO}" --token "${Env:RUNNER_TOKEN}" --ephemeral --work _work --labels "${Env:RUNNER_LABEL},cdkghr:started:$(Get-Date -UFormat %s)" ${RunnerFlags} --name "${Env:RUNNER_NAME}"',
+        'if ($RUNNER_LEVEL -eq "org") { $REGISTRATION_URL = "https://\${githubDomainPath}/\${ownerPath}" } elseif ($RUNNER_LEVEL -eq "repo") { $REGISTRATION_URL = "https://\${githubDomainPath}/\${ownerPath}/\${repoPath}" } else { Write-Host "Invalid runnerLevel: $RUNNER_LEVEL"; return 1 }',
```

Comment:
It might be easier to build the registration URL in token retriever and pass it in the step function. This way we don't have to add `if`/`else` everywhere.

Reply:
argh - you`re damn right - that would make it way easier ... will change this when I have time...
I tested the github app on github.com with a organization and runners registered on org level. 
Yes it works quite well.

<img width="373" alt="image" src="https://github.com/CloudSnorkel/cdk-github-runners/assets/5619511/005dc271-2634-4b11-b771-07af8d8b61bb">